src/optim_problems/opt_cep.jl

```diff
@@ -129,26 +151,31 @@ function setup_opt_cep_lost_emission!(cep::OptModelCEP,
   ## LOST EMISSION ##
   # Lost Emission variable #
   push!(cep.info,"Variable LE[impact_{environment}] ≥ 0 in kg")
-  @variable(cep.model, LE[impact=["CO2"]] >=0)
+  @variable(cep.model, LE[impact=set["impact"][2:end]] >=0)
   return cep
 end
 
 """
-     setup_opt_cep_fix_design_variables!(cep::OptModelCEP, ts_data::ClustData, opt_data::OptDataCEP) set::Dict)
+     setup_opt_cep_fix_design_variables!(cep::OptModelCEP, ts_data::ClustData, opt_data::OptDataCEP, fixed_design_variables::Dict{String,Any})
 Fixing variables CAP based on first stage vars
 """
 function setup_opt_cep_fix_design_variables!(cep::OptModelCEP,
                                   ts_data::ClustData,
-                                  opt_data::OptDataCEP;
-                                  fixed_design_variables::Dict{String,OptVariable}=Dict{String,OptVariable}())
+                                  opt_data::OptDataCEP,
+                                  fixed_design_variables::Dict{String,Any})
```

Comment:
that's a little annoying. Using Dict{String,OptVariable} errors in JLD2, because the lookup has a weird Vararg{} Type...